test/controllers/csvs_controller_test.rb

```diff
@@ -0,0 +1,335 @@
+require_relative '../test_helper'
+
+describe CsvsController do
+  let(:deployer) { users(:deployer) }
+  let(:export) { csv_exports(:pending) }
+
+  describe "permissions test" do
+    as_a_admin do
+      describe "a GET to new" do
+        it "renders the page with user link & admin menu" do
+          get :new
+          assert_template :new
+          assert @response.body.include? "Create Deploys CSV Report"
+          assert @response.body.include? "Download Users CSV Report"
+          assert @response.body.include? "Admin"
+          assert @response.body.include? "Environment variables"
+        end
+      end
+    end
+
+    as_a_deployer do
+      describe "a GET to new" do
+        it "renders the page without user link & limited admin menu" do
+          get :new
+          assert_template :new
+          assert @response.body.include? "Create Deploys CSV Report"
+          refute @response.body.include? "Download Users CSV Report"
+          assert @response.body.include? "Admin"
+          refute @response.body.include? "Environment variables"
```

Comment:
Checking that a keyword in the admin menu options (upper set) only displays for admins, since reporting of deploys may be done by someone that isn't an admin.

If there is a different thought on the permissions for reporting, it could be limited to only admins.  Ideally a permission level other than admin should be able to generate both the users and deploys reports so people that need those reports don't have to be admins.  After this only the user report will require admin, but that could probably be changed.